index.html

```diff
@@ -0,0 +1,857 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <link
+      rel="shortcut icon"
+      href="./images/the-next-web-tnw-logo.png"
+      type="image/x-icon"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css"
+    />
+    <title>TNW</title>
+  </head>
+  <body>
+    <nav class="navigation">
+      <div class="black-nav-links">
+        <div class="nav-left">
+          <ul class="ul-black-nav">
+            <li><a href="#">News</a></li>
+            <li><a href="#">Events</a></li>
+            <li><a href="#">Business</a></li>
+            <li><a href="#">Amas</a></li>
+            <li><a href="#">Spaces</a></li>
+            <li><a href="#">Terms & Condition</a></li>
+          </ul>
+        </div>
+        <div class="nav-right">
+          <ul class="ul-black-nav">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Advertise</a></li>
+            <li><a href="#">Jobs</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="fix-nav">
+        <div class="fix-nav-left">
+          <img class="logo" src="images/the-next-web-tnw-logo.png" alt="logo" />
+          <!-- ++++++++++++++++++++++++++++++ Displays on Ipad and Desktop+++++++++++++++++++++++++  -->
+          <div class="white-nav-links">
+            <ul class="ul-white-nav">
+              <li><a href="#">latest</a></li>
+              <li><a href="#">hard fork</a></li>
+              <li><a href="#">plugged</a></li>
+              <li><a href="#">readme</a></li>
+              <li><a href="#">growth quateres</a></li>
+              <li><a href="#">neural</a></li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="fix-nav-right">
+          <div class="search-img">
+            <i class="fas fa-search"></i>
+          </div>
+          <div class="hamburger">
+            <i class="fas fa-bars"></i>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <div class="sweatpants">
+      We heard you like events and sweatpant so we made event ...
+      <span class="disappear">you can join in your sweatpant ➔</span>
+    </div>
+
+    <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        Main Section
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+    <main class="main-section">
+      <div class="main-wrapper">
+        <div class="header-img hero-img-1 span-2-rows desk-header-img">
+          <div class="overlay">
+            <p class="img-text text-bg">
+              Netflix will cancel account if you havent used it in a while
```

Comment:
- [ ] Kindly add the logo to the main article for large screens. The second article doesn't have a gradient. Kindly fix that. 

- [ ] For medium and small screens, the gradient should be there. The text isn't displayed for the articles with gradients and some text on the navbar is hidden. 

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83333599-eaed3700-a266-11ea-976f-9d0157170fd7.png) | ![image](https://user-images.githubusercontent.com/57414789/83333583-d6a93a00-a266-11ea-8014-8a14756d3c09.png)
![image](https://user-images.githubusercontent.com/57414789/83333637-2d167880-a267-11ea-8c59-bd7bbcf42156.png) | ![image](https://user-images.githubusercontent.com/57414789/83333660-415a7580-a267-11ea-8273-ad67367b0fcb.png)